src/ceph_fuse.cc

```diff
@@ -81,9 +81,10 @@ static void fuse_usage()
 void usage()
 {
   cout <<
-"usage: ceph-fuse [-n client.username] [-m mon-ip-addr:mon-port] <mount point> [OPTIONS]\n"
-"  --client_mountpoint/-r <sub_directory>\n"
-"                    use sub_directory as the mounted root, rather than the full Ceph tree.\n"
+"\nusage: ceph-fuse [-n client.username] [--client_fs <fsname>] [-m mon-ip-addr:mon-port] [--client_mountpoint/-r <sub_directory>] <mount point> [OPTIONS]\n\n"
```

Comment:
Perhaps we can re-arrange the order here to bring `--client_fs` and `--client_mountpoint` next to each other since often these might be used together. Grouping this makes is it easier for readers to remember the related options.

Reply:
Done